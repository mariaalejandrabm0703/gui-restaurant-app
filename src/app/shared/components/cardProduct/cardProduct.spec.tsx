import '@testing-library/jest-dom';

import { cleanup, render, wait } from '@testing-library/react';
import { mount, shallow } from 'enzyme';
import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import CardProduct from './cardProduct';

afterEach(cleanup);

it('rederizar el componente CardInformation sin errores en modo detalle', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <CardProduct />,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });