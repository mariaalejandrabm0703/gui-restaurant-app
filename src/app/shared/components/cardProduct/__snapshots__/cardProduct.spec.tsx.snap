// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Compara snapshot del componente CardProduct 1`] = `
<div
  className="card card-div"
>
  <img
    alt="producto prueba"
    className="img-fluid img-cart"
    src="https://cdns.iconmonstr.com/wp-content/assets/preview/2019/240/iconmonstr-product-3.png"
  />
  <div
    className="card-body d-flex"
  >
    <div
      className="row"
    >
      <div
        className="col-12 card-info"
      >
        <p
          className="card-title"
        >
          producto prueba
        </p>
        <p
          className="card-text"
        >
          $500.00
        </p>
      </div>
      <div
        className="col-12"
      >
        <div
          className="btn-cart card-buton"
        >
          <a
            className="btn btn-warning card-buton-item"
            onClick={[Function]}
            type="submit"
          >
            Agregar 
            <i
              className="fas fa-shopping-cart"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
